### ISSUE TITLE
 Uncaught Error: Access to undeclared static property: Phalcon\Di::$_default

### ISSUE THREAD

**Issue by wydhit** [phalcon/cphalcon#12958]
build project by     `phalcon project --type=modules`

in  `IndexController `    `indexAction`
```php
for ($i=1;$i<2000;$i++){
        file_get_contents(__DIR__.'/IndexController.php');
        }
        return 'test';
```

then I visit my website faster;   Fatal error   show;

but visit my website slowly ;  Fatal error not   show;
```
Fatal error: Uncaught Error: Access to undeclared static property: Phalcon\Di::$_default in E:\mytest\PhalconAdmin\admin\public\default\app\bootstrap_web.php:17 
Stack trace:
#0 [internal function]: Phalcon\Di->__construct() 
#1 E:\mytest\PhalconAdmin\admin\public\default\app\bootstrap_web.php(17): Phalcon\Di\FactoryDefault->__construct()
#2 E:\mytest\PhalconAdmin\admin\public\default\public\index.php(2): require('E:\\mytest\\Phalc...') 
#3 {main} Next Error: Access to undeclared static property: Phalcon\Di::$_default in E:\mytest\PhalconAdmin\admin\public\default\app\bootstrap_web.php:17 Stack trace:

#0 [internal function]: Phalcon\Di->__construct() 
#1 E:\mytest\PhalconAdmin\admin\public\default\app\bootstrap_web.php(17): Phalcon\Di\FactoryDefault->__construct() 
#2 E:\mytest\PhalconAdmin\admin\public\default\public\index.php(2): require('E:\\mytest\\Phalc...') 
#3 {main} thrown in E:\mytest\PhalconAdmin\admin\public\default\app\bootstrap_web.php on line 17
```

* Phalcon version: (`3.2.0`)
* PHP Version: (`PHP Version 7.1.7`)
* Operating System:window10
* Server:  Apache
* Server API | Apache 2.0 Handler
*php  opcache.enable=0

**Comment by Jurigag** [comment]
Duplicate of https://github.com/phalcon/cphalcon/issues/12056

**Comment by wydhit** [comment]
switch to php5.6  ，   I've had no problems.

**Comment by Jurigag** [comment]
There is solution provided there - switch to php-cgi + nginx/apache, and problem solved, even with php 7.1

**Comment by sergeyklay** [comment]
Close in favor of #12056
